checkstyle.json

```diff
@@ -517,6 +517,22 @@
 		},
 		{
 			"props": {
+				"severity": "INFO",
+				"ignorePackages": [
```

Comment:
yeah, I should probably change it to `ignoreModules`

Additional types defined in an imported module are hard to detect, because we have no info on them. We could create some kind of a cache, containing a map of files and types, but that wouldn't solve all issues:
- some files may still be in our queue when they are referenced by files earlier in checkstyle's file list
- modules from libs outside of `-s` are not processed, so we don't know which additional types might be in there